src/node/rpc/json_handler.h

```diff
@@ -128,6 +132,46 @@ namespace ccf
       return packing.value_or(serdes::Pack::Text);
     }
 
+    static serdes::Pack get_response_pack(
+      const std::shared_ptr<enclave::RpcContext>& ctx,
+      serdes::Pack request_pack)
+    {
+      serdes::Pack packing = request_pack;
+
+      const auto accept_it = ctx->get_request_header(http::headers::ACCEPT);
+      if (accept_it.has_value())
+      {
+        const auto& accept = accept_it.value();
+        if (accept == http::headervalues::contenttype::JSON)
+        {
+          packing = serdes::Pack::Text;
+        }
+        else if (accept == http::headervalues::contenttype::MSGPACK)
+        {
+          packing = serdes::Pack::MsgPack;
+        }
+        else if (accept == "*/*")
+        {
+          packing = request_pack;
+        }
+        else
+        {
+          throw RpcException(
+            HTTP_STATUS_NOT_ACCEPTABLE,
+            ccf::errors::UnsupportedContentType,
+            fmt::format(
+              "Unsupported content type {} in accept header. Only {} and {} "
+              "are currently "
```

Comment:
That's a weird line break, but fine.

Reply:
Yeah, clang-format did that. Looks like it can introduce line breaks to satisfy the rules, but it doesn't re-format the whole string. I fixed it.